diff --git a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt
index 9e28538..7633a14 100644
--- a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt
+++ b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStack.kt
@@ -297,7 +297,7 @@ class ScreenStack(context: Context?) : ScreenContainer(context) {
         drawingTime: Long
     ): Boolean {
         if (mRemovalTransitionStarted) {
-            mDrawingOp!!.draw(canvas)
+            canvas?.let { mDrawingOp!!.draw(it) }
             return true
         }
         return super.drawChild(canvas, child, drawingTime)
diff --git a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt
index e8b6fa7..b7b40ba 100644
--- a/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt
+++ b/node_modules/react-native-screens/android/src/main/java/com/swmansion/rnscreens/ScreenStackHeaderConfig.kt
@@ -390,7 +390,7 @@ class ScreenStackHeaderConfig(context: Context) : ViewGroup(context) {
         // we want to apply theme attrs while keeping close and search icon tint
         TypedValue().let { typedValue ->
             theme.resolveAttribute(
-                androidx.appcompat.R.attr.toolbarNavigationButtonStyle, typedValue, true
+                androidx.appcompat.R.attr.navigationIcon, typedValue, true
             )
             navView.setBackgroundResource(typedValue.resourceId)
         }